fdbserver/include/fdbserver/BulkLoadUtil.actor.h

```diff
@@ -28,11 +28,18 @@
 #include "fdbclient/BulkLoading.h"
 #include "flow/actorcompiler.h" // has to be last include
 
-// Get the bulkLoadTask metadata of the dataMoveMetadata since the minVersion given the dataMoveId
-ACTOR Future<Optional<BulkLoadTaskState>> getBulkLoadTaskStateFromDataMove(Database cx,
-                                                                           UID dataMoveId,
-                                                                           Version minVersion,
-                                                                           UID logId);
+// Erase file folder
+void clearFileFolder(const std::string& folderPath, const UID& logId = UID(), bool ignoreError = false);
```

Comment:
Doesn't belong in platform?

Reply:
Currently, let's make this specific to bulkLoad. We will move this function to a background thread.